lcli/src/main.rs

```diff
@@ -409,15 +405,13 @@ fn main() {
                         .value_name("INTEGER")
                         .takes_value(true)
                         .help("The gas limit field in the execution payload generated.")
-                        .required(true)
                         .default_value("30000000"),
                 )
                 .arg(
                     Arg::with_name("file")
                         .long("file")
                         .value_name("FILE")
                         .takes_value(true)
-                        .required(true)
```

Comment:
There's no `default_value` for this one - we may want to keep this one or assign a default value here.

Reply:
Good catch!